proofs/cbmc/crypto_sign_verify_internal/Makefile

```diff
@@ -0,0 +1,66 @@
+# Copyright (c) The mldsa-native project authors
+# SPDX-License-Identifier: Apache-2.0 OR ISC OR MIT
+
+include ../Makefile_params.common
+
+HARNESS_ENTRY = harness
+HARNESS_FILE = crypto_sign_verify_internal_harness
+
+# This should be a unique identifier for this proof, and will appear on the
+# Litani dashboard. It can be human-readable and contain spaces if you wish.
+PROOF_UID = crypto_sign_verify_internal
+
+DEFINES +=
+INCLUDES +=
+
+REMOVE_FUNCTION_BODY +=
+UNWINDSET +=
+
+PROOF_SOURCES += $(PROOFDIR)/$(HARNESS_FILE).c
+PROJECT_SOURCES += $(SRCDIR)/mldsa/sign.c
+
+CHECK_FUNCTION_CONTRACTS=$(MLD_NAMESPACE)crypto_sign_verify_internal
+USE_FUNCTION_CONTRACTS=$(MLD_NAMESPACE)unpack_pk $(MLD_NAMESPACE)unpack_sig $(MLD_NAMESPACE)polyvecl_chknorm\
+                       $(FIPS202_NAMESPACE)shake256 $(FIPS202_NAMESPACE)shake256_init \
```

Comment:
I think you need to add mld_H (no prefix) to this list.

Reply:
fixed.